hera_pspec/pspecdata.py

```diff
@@ -56,17 +60,24 @@ def __init__(self, dsets=[], wgts=None, dsets_std=None, labels=None,
 
         cal_flag : bool, optional
             If True, propagate flags from calibration into data
+
+        r_cache : dict, optional
```

Comment:
r_cache isn't actually a parameter in the class call

Reply:
removed this.